include/deal.II/lac/trilinos_precondition.h

```diff
@@ -153,6 +153,12 @@ namespace TrilinosWrappers
                          const dealii::parallel::distributed::Vector<double> &src) const;
 
     /**
+     * Return a shared pointer to the underlaying Trilinos Epetra_Operator.
+     * So you can use the preconditioner with unwrapped Trilinos solver.
+     */
+    std_cxx11::shared_ptr<Epetra_Operator> trilinos_operator() const;
```

Comment:
I'd say just return a reference to the object itself, rather than the shared pointer.